x/staking/types/validator.go

```diff
@@ -400,14 +401,42 @@ func (v Validator) DelegatorShareExRate() sdk.Dec {
 	return sdk.NewDecFromInt(v.Tokens).Quo(v.DelegatorShares)
 }
 
-// Get the bonded tokens which the validator holds
+// get the bonded tokens which the validator holds
 func (v Validator) BondedTokens() sdk.Int {
 	if v.Status == sdk.Bonded {
 		return v.Tokens
 	}
 	return sdk.ZeroInt()
 }
 
+// get the Tendermint Power
+// a reduction of 10^9 from validator tokens is applied
+func (v Validator) TendermintPower() int64 {
+	if v.Status == sdk.Bonded {
+		return v.PotentialTendermintPower()
+	}
+	return 0
+}
+
+var powerReduction = new(big.Int).Exp(big.NewInt(10), big.NewInt(9), nil)
```

Comment:
We should probably implement `Int#Exp` and `Uint#Exp`. Also, why not just wrap this in a `sdk.NewIntFromBigInt`?

Reply:
Yeah I think this makes sense - going to open up an issue and perform this separately (will also perform an `types/int` cleanup simultaneously)  https://github.com/cosmos/cosmos-sdk/issues/3471